WorkflowStreamer/src/main/java/com/workflowstreamer/manager/UsersManager.java

```
@@ -45,9 +47,18 @@ public Response addNewUser(ImmutableNewUser newUser) {
             int userId = usersDao.insertUser(newUser.getEmail(), newUser.getUsername(), newUser.getPassword());
             ImmutableUser insertedUser = usersDao.getUserById(userId);
             response = Response.ok(insertedUser);
+            addDefaultStages(insertedUser.getUserId());
         } catch (UnableToExecuteStatementException e) {
             response = Response.status(Response.Status.FORBIDDEN);
         }
         return response.build();
     }
+
+    private void addDefaultStages(int userId) {
+        try {
+            DEFAUL_STAGES.forEach(stage -> usersDao.insertUserStage(userId, stage));
+        } catch (UnableToExecuteStatementException e) {
+            e.printStackTrace();
```

This can be a silent error for now as it's rare to happen and shouldn't effect new user creation. If this error is not caaught here, a new user will be added, but an error will be returned in the response.